anthem/lyrics/loaders.py

```diff
@@ -100,3 +100,13 @@ def load_csv_stream(ctx, model, data,
             rows = list(_rows)
         if rows:
             load_rows(ctx, model, header, rows)
+
+
+def update_translations(ctx, modules):
+    """ Update translations from module list
+
+    :param modules: a list of modules
+    """
+    for module in modules:
+        ctx.env['ir.module.module'].with_context(overwrite=True).search(
+            [('name', '=', module)]).update_translations()
```

Comment:
Thanks, this is great, but should not be in the `loaders` which should be the place for lyrics that load records (using csv, xml, yaml or whatever).

I think we should have a module `modules.py` which contains `update_translations` and `uninstall` (which should be moved from `uninstaller.py` but  `uninstaller.py` should be kept with a import of `uninstall` for backward compat).

Reply:
@leemannd I just notice now, this code is still there :'-( (and in modules.py as well)

Reply:
also, wouldn't be better since you gonna touch it ;P to do `search([('name', 'in', modules)]).update_translations()` w/out looping?

Reply:
aaaaaaaaand support `filter_lang=None` to update just one lang when needed